clifford/numba/_multivector.py

```diff
@@ -250,6 +250,20 @@ def impl(a, b):
         return impl
 
 
+@numba.extending.overload(operator.pow)
+def ga_pow(a, b):
+    if isinstance(a, MultiVectorType) and isinstance(b, types.Integer):
+        gmt_func = a.layout_type.obj.gmt_func
+        def impl(a, b):
+            if b == 0:
```

Comment:
What about when `b < 0`? For now, `raise NotImplementedError` is probably fine.

Looks like `MultiVector.__pow__` is broken too.